src/content/docs/ecosystem/tachyonfx.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Effects and animations (Tachyonfx)
+sidebar:
+  order: 1
+---
+
+> An effects and animation library for Ratatui applications.
+
+Build complex animations by composing and layering simple effects, bringing smooth transitions and
+visual polish to the terminal.
+
+![](https://github.com/junkdog/tachyonfx/raw/development/images/demo-0.6.0.gif)
```

Comment:
as a demo for tachyonfx, demo-0.6.0 is pretty slow before it gets interesting. don't have anything up-to-date, but exabind packs its punch in the first couple of seconds, so i suggest changing the link to

```
![](https://github.com/junkdog/exabind/raw/main/demo/exabind-02.gif)
```

Reply:
maybe a link to exabind's ratzilla incarnation too: https://junkdog.github.io/exabind/